### PR TITLE
Harmony: Table

### LINKED ISSUE #748: Harmony: Table

### PR DESCRIPTION
**Pull request by DenisVorop**
## PR includes

- [x] Bug Fix
- [x] Feature

## Related issues

Resolve #748 
Related issues https://github.com/taskany-inc/issues/pull/2228

## QA Instructions, Screenshots, Recordings

Снимок экрана 2024-01-29 в 18 00 52

Снимок экрана 2024-01-29 в 18 01 17

### Collapsible table
Снимок экрана 2024-01-29 в 18 01 35

### CODE REVIEW

File: src/harmony/index.ts [hunk 1]
```diff
@@ -1,11 +1,22 @@
+export * from './AutoComplete/AutoComplete';
```
Comment (awinogradov) [inline src/harmony/index.ts:1]: Unnecessary diff

File: src/harmony/Table/Table.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+                {data.map((row) => (
+                    <TableRow
+                        style={{
+                            padding: '8px 12px',
+                            background: 'var(--layer)',
+                            border: '1px solid var(--gray-900)',
+                            borderRadius: 'var(--radius-m)',
+                        }}
+                    >
+                        {row.map(({ render, name, style, width, item, head: _, ...rest }, index) => (
```
Comment (awinogradov) [inline src/harmony/Table/Table.stories.tsx:159]: Unused variable

**Review by awinogradov — CHANGES_REQUESTED** [review]

File: src/harmony/Table/Table.module.css [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+.Row {
+    display: flex;
+    align-items: center;
+    gap: var(--gap-m);
+}
+
+.Cell {
+    display: flex;
+    flex-shrink: 0;
+    width: var(--table-cell-width);
```
Comment (LamaEats) [inline src/harmony/Table/Table.module.css:15]: Add fallback for variable value, ex. `auto` or `unset`

**Review by LamaEats — APPROVED** [review]

**Review by awinogradov — APPROVED** [review]

**Pull request merged**